api/setup.py

```diff
@@ -9,6 +9,7 @@
 import sys
 
 import itertools
+from setuptools import find_packages
```

Comment:
Perhaps, a good thing to do now might be to support both `distutils` and `setuptools`. If a person doesn't have `setuptools`, we default to `disutils`. In that case we cannot use `find_packages`, so it might still be worth it to make a list of all packages we want to be included (everything in `bots/*` and `bots_api`) and supply that to `setup()` instead.